### DISCUSSION TITLE
[@types/node] process.getuid should be unioned with `undefined`

### DISCUSSION THREAD

**Discussion by Josh-Cena** [DefinitelyTyped/DefinitelyTyped#60494] [Issues with a @types package]
From the [docs](https://nodejs.org/dist/latest-v18.x/docs/api/process.html#processgetuid), it looks like `process.getuid` would be `undefined` on non-Unix environments.

I don't have a Windows machine to test, but I did see a few SO questions about "`process.getuid` is not a function".

[a comment by an automated account is not shown]

**Reply by mcollina** [reply]
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Reply by Josh-Cena** [reply]
Ah, sure! I can take a look later. This can be released alongside `@types/node@18` as a BC, right?

**Reply by mcollina** [reply]
I hope so
